### PR TITLE
Fix issue #1353

### PR DESCRIPTION
**Pull request by vuphin**
remove only a line to fix

### CODE REVIEW

**Comment by johnthagen** [comment]
Cross referencing issue #1353

**Review by jonadem — COMMENTED** [review]
Is commit 0f5224954f5a47363d6339317c9317515f58bcca really needed ?

At my side, it doesn't want to compile with this.
I removed this duplicated line and afterwards, it builded just fine.

**Comment by jonadem** [comment]
This is a file from Autocad opened with the code from master:
LC_master
This is with the pull request:
LC_vuphin

This is a great improvement !

**Comment by pollywinters** [comment]
Applying this pull request, I find that there is an error produced in file librecad/src/src.pro, as per the comment by [jonadem](https://github.com/jonadem) above dated 26 Jun 2021.
The line `lib/gui/rs_linetypepattern.cpp \` becomes duplicated, resulting in build failure.
Removing the duplicate of the line allows the build to complete. The sample DXF file provided by [johnthagen](https://github.com/johnthagen) will then open and display correctly.
Note added to issue #1353

File: librecad/src/lib/engine/rs_hatch.cpp [hunk 1]
```diff
@@ -607,7 +607,7 @@ void RS_Hatch::draw(RS_Painter* painter, RS_GraphicView* view, double& /*pattern
 //                    if (! (pa.size()>0 && (pa.last() - pt1).manhattanLength()<=2)) {
 //                        jp<<pt1;
 //                    }
-                    if(pa.size() && (pa.last()-pt1).manhattanLength()>=1)
+                    //if(pa.size() && (pa.last()-pt1).manhattanLength()>=1)
```
Comment (lordofbikes) [inline librecad/src/lib/engine/rs_hatch.cpp:610]: This condition should avoid to store points twice. A container containing a bunch of lines have end points matching with the start point of the following entity. Storing all points will double the size of the resulting polygon.
So the condition has its right to exist, but it failed on the very first start point which led to the issue.

**Comment by lordofbikes** [comment]
There was too many noise in this PR, 5 files changed for 1 line removed. And it also contained an error in the project file.
Also on reviewing I found out, that removing the if condition was not the correct solution.
So I fixed it now with https://github.com/LibreCAD/LibreCAD/commit/5480eba6af927db91ca43338815b40fdd4d3130b.

Many thanks anyhow for pointing this out and your effort to battle it down.

**Pull request closed without merging**